Mathlib/Algebra/Group/Hom/Instances.lean

```diff
@@ -19,56 +22,74 @@ operations.
 Finally, we provide the `Ring` structure on `AddMonoid.End`.
 -/
 
-assert_not_exists AddMonoidWithOne Ring
```

Comment:
#27319 should make this unecessary.

Reply:
Does it also make the import bumps negligible?

Reply:
You tell me. Resolving this comment thread since the `assert_not_exists` is now untouched.